influxdb/client.py

```diff
@@ -98,6 +100,13 @@ def __init__(self,
         self._database = database
         self._timeout = timeout
         self._retries = retries
+        if compression is True:
+            self._compression = 9
+        else:
+            self._compression = compression
+
+        assert self._compression is False or \
```

Comment:
Why should we blow up here if we don't match one of the values? This should be set to a proper default and move along without any issue for the user.

Reply:
Hmm, ok, will do. Usualy I prefer my code to blow up instead of hiding bugs ;)

Reply:
If you flip this to be two parameters as outline above then this should give you enough assurance that things are working as expected.